bin/rspecq

```diff
@@ -5,6 +5,7 @@ require "rspecq"
 DEFAULT_REDIS_HOST = "127.0.0.1"
 DEFAULT_REPORT_TIMEOUT = 3600 # 1 hour
 DEFAULT_MAX_REQUEUES = 3
+DEFAULT_FAIL_FAST = 1
```

Comment:
Do we want to enable this by default? I'd say no.

Reply:
966619b